src/main/java/uk/gov/ons/ctp/response/action/service/ActionProcessingService.java

```diff
@@ -93,14 +93,19 @@ public void processActionRequest(final Action action) throws CTPException {
       final ActionRequest actionRequest = prepareActionRequest(action);
 
       if (actionRequest != null) {
-        actionInstructionPublisher.sendActionInstruction(actionType.getHandler(), actionRequest);
+        if (actionType.getHandler().equals("Field")
+            && (actionRequest.getCaseGroupStatus().equals("NOTSTARTED")
+                || actionRequest.getCaseGroupStatus().equals("INPROGRESS"))) {
+          actionInstructionPublisher.sendActionInstruction("Field", actionRequest);
+        } else {
+          actionInstructionPublisher.sendActionInstruction(actionType.getHandler(), actionRequest);
```

Comment:
This will still send completed cases to field

Reply:
Fixed, but I think it would be good to test this scenario since it's such a key aspect of the feature.